.nais/naiserator-dev.yaml

```diff
@@ -74,3 +74,5 @@ spec:
       value: "dev-gcp.teamsykefravr.istilgangskontroll"
     - name: ISTILGANGSKONTROLL_URL
       value: "http://istilgangskontroll"
+    - name: SENOPPFOLGING_SVAR_CONSUMER_ENABLED
+      value: "true"
```

Comment:
Burde vi vente til vi kan lagre?

Reply:
Ja, jeg deployet til dev for å få teste at konsumeringen fungerte, men kan godt skru den av enn så lenge :+1: